From 0d562cb2e5a4e330296a4cdf12aa555435f4b850 Mon Sep 17 00:00:00 2001
From: Michael Goffioul <michael.goffioul@lincor.com>
Date: Wed, 28 Apr 2021 16:10:14 -0400
Subject: [PATCH 2/2] Rename libgbm module to libmesa_gbm

AOSP minigbm now defines its own libgbm module as a shared library,
which conflicts with the one in mesa. This is accompanied by a similar
change in gbm_gralloc.
---
 src/gbm/Android.mk | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/gbm/Android.mk b/src/gbm/Android.mk
index 1fe56524322..b432983eb9f 100644
--- a/src/gbm/Android.mk
+++ b/src/gbm/Android.mk
@@ -36,7 +36,7 @@ LOCAL_STATIC_LIBRARIES := \
 	libmesa_util \
 	libmesa_loader
 
-LOCAL_MODULE := libgbm
+LOCAL_MODULE := libmesa_gbm
 
 LOCAL_SRC_FILES := \
 	$(gbm_core_FILES) \
-- 
2.30.2

